windshaft/Dockerfile

```diff
@@ -1,31 +1,25 @@
-# Windshaft dependencies are for Ubuntu
-FROM ubuntu:vivid
+FROM node:0.10-slim
 
-MAINTAINER Azavea
+MAINTAINER Azavea <systems@azavea.com>
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
     build-essential \
-    nodejs \
-    npm \
     git-core \
     libcairo2-dev \
     libpango1.0-dev \
-    libjpeg8-dev \
+    libjpeg62-turbo-dev \
     libgif-dev \
     libpq-dev \
     && rm -rf /var/lib/apt/lists/*
 
-# link nodejs command to node
-RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
-
 RUN mkdir -p /opt/windshaft
 WORKDIR /opt/windshaft
 COPY . /opt/windshaft
 
 # NB: the most recent versions do not work with the provided configurations
-RUN npm install windshaft@0.36.0
+RUN npm install --unsafe-perm windshaft@0.36.0
```

Comment:
Unclear. My suspicion is that it's related to installing Node via Ubuntu's repositories. Given what `unsafe-perm` is supposed to prevent, seems like it should be happening with both base images.